rotkehlchen/tests/unit/decoders/test_makerdao_sai.py

```diff
@@ -499,75 +1176,328 @@ def test_makerdao_sai_underlying_collateral_removal(database, ethereum_inquirer)
 
 
 @pytest.mark.parametrize('ethereum_accounts', [['0x6D1723Af1727d857964d12f19ed92E63736c8dA2', '0x720972Dc53741a72fEE22400828122836640a74b']])  # noqa: E501
-def test_makerdao_sai_proxy_interaction(database, ethereum_inquirer):
+def test_makerdao_sai_proxy_interaction(ethereum_transaction_decoder):
     """
     Data for proxy interaction is taken from
     https://etherscan.io/tx/0xf4203a8b507b0b382903bd8d35dcff29aea98de76b89f745d94705d54b67646f
     """
     tx_hex = '0xf4203a8b507b0b382903bd8d35dcff29aea98de76b89f745d94705d54b67646f'
     evmhash = deserialize_evm_tx_hash(tx_hex)
-    events, _ = get_decoded_events_of_transaction(
-        ethereum_inquirer=ethereum_inquirer,
-        database=database,
+    transaction = EvmTransaction(
+        chain_id=ChainID.ETHEREUM,
         tx_hash=evmhash,
+        timestamp=1565146195,
+        block_number=8300924,
+        from_address='0x6D1723Af1727d857964d12f19ed92E63736c8dA2',
```

Comment:
One small comment for this and all the other tests. We are using variables for the addresses to avoid repeating the same string over the test and also to make them easier to read. You can see an example here https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/unit/decoders/test_compound.py#L14

For each test replace all the appearance of relevant address